convert2rhel/breadcrumbs.py

```diff
@@ -225,6 +225,14 @@ def print_data_collection(self):
             RHSM_CUSTOM_FACTS_FILE,
         )
 
+        # Add in the warning about the system back up here!
+        loggerinst.warnign(
+            "Convert2RHEL modifies the systems during the analysis and then rolls back these"
+            " changes when the analysis is complete. In rare cases, this rollback can fail.\n"
+            "By continuing, you confirm that you have made a system backup and verified that"
+            " you can restore from the backup."
+        )
+
         utils.ask_to_continue()
```

Comment:
I think we should move this message to be printed right after we call this function at line 328. 

The reason for that is this message is mentioning analysis and rollback, and to tell the user to back up their systems. The function itself is about data collection. They are not correlated.

Reply:
Gotcha yeah I wasn't sure if I should include it else where or there as I was thinking that it might be a little redundant to ask the user to continue twice back to back. I can definitely get that switch to after the function!